recipes/zstd/all/conanfile.py

```diff
@@ -96,6 +90,6 @@ def package_info(self):
         self.cpp_info.components["zstdlib"].names["cmake_find_package"] = zstd_cmake
         self.cpp_info.components["zstdlib"].names["cmake_find_package_multi"] = zstd_cmake
         self.cpp_info.components["zstdlib"].set_property("cmake_target_name", "zstd::{}".format(zstd_cmake))
-        self.cpp_info.components["zstdlib"].libs = tools.collect_libs(self)
+        self.cpp_info.components["zstdlib"].libs = files.collect_libs(self)
```

Comment:
Might be worth using explicit list 👍

Reply:
I agree! I think it's just the one file regardless,  will check!

Reply:
https://github.com/facebook/zstd/blob/dev/build/cmake/lib/CMakeLists.txt#L113-L136

don't forget to properly handle clang-cl

Reply:
```suggestion
        self.cpp_info.components["zstdlib"].libs = files.collect_libs(self) # TODO: Be explicit https://github.com/conan-io/conan-center-index/pull/12010#discussion_r936371452
```